packages/alfa-act/src/interview.ts

```diff
@@ -9,34 +9,35 @@ import { Rule } from "./rule";
  * As `Interview` is a recursive type that models nested chains of `Question`s,
  * we need to limit the depth to which this recursion can happen to avoid the
  * TypeScript compiler going into an infinite type resolution loop. This is done
- * by maintaining a pointer, `D extends number` that down to -1 at which point
- * the recursion will stop. Given a pointer `D`, the `Depths` tuple provides
- * `D - 1 = Depths[D]`.
+ * by maintaining a pointer, `D extends number` that counts down to -1 at which
+ * point the recursion will stop. Given a pointer `D`, the `Depths` tuple
+ * provides `D - 1 = Depths[D]`.
  */
 type Depths = [-1, 0, 1, 2];
 
 /**
  * @public
  */
-export type Interview<Q, S, T, D extends number = 3> =
-  | T
+export type Interview<Q, S, C, A, D extends number = 3> =
```

Comment:
Same remark here, `C` and `S` are usually the same, but we can't default without swapping parameter order 😞